pages/aboutus.js

```diff
@@ -0,0 +1,148 @@
+import HowItWorks from "./../components/Aboutus/HowItWorks";
+import TeamProfile from "./../components/Aboutus/TeamProfile";
+import WhyChooseUs from "./../components/Aboutus/WhyChooseUs";
+import Header from "./../components/Header";
+
+import Rebaz from "../assets/TeamPic/Rebaz.jpg";
+import Aya from "../assets/TeamPic/aya.jpg";
+import Zahraa from "../assets/TeamPic/zahraa.jpg";
+import Bnar from "../assets/TeamPic/bnar.jpg";
+import Lara from "../assets/TeamPic/Lara.jpg";
+
+import { BsFillFileEarmarkTextFill } from "react-icons/bs";
+import { BsFillPencilFill } from "react-icons/bs";
+import { FaHandshake } from "react-icons/fa";
+
+import { FaPenNib } from "react-icons/fa";
+import { FaBookmark } from "react-icons/fa";
+import { FaFileUpload } from "react-icons/fa";
+import { FaCheckDouble } from "react-icons/fa";
+
+import HeadPic from "../assets/img_about.png";
+import SectionPic from "../assets/img_about2.png";
+
+const Sec1 = [
+  {
+    title: "Create your resume",
+    desc: "creating your resume helps you apply faster and showcase your experience and skills to employers",
+    logo: BsFillFileEarmarkTextFill,
+  },
+  {
+    title: "Apply for a Job",
+    desc: "Through our website you can find different kind of jobs from different tech categories and applying is made easy once you set up your resume",
+    logo: BsFillPencilFill,
+  },
+  {
+    title: "Get hired now",
+    desc: "Once you apply for a job the employer will be able to see your resume and the position you applied for, which makes hiring easy",
+    logo: FaHandshake,
+  },
+];
+
+const Sec2 = [
+  { title: "Posted", subtitle: "15k Vacancies", logo: FaPenNib },
+  { title: "Saved", subtitle: "5k Vacancies", logo: FaBookmark },
+  { title: "Submitted", subtitle: "12k Vacancies", logo: FaFileUpload },
+  { title: "Hired", subtitle: "8k Vacancies", logo: FaCheckDouble },
+];
+
+export default function aboutus() {
+  return (
+    <div className="bg-body">
+      <Header title="Find Your Job" subtitle="Live Your Dream " img={HeadPic} />
```

Comment:
the header image is too small and too much empty space between text and image

Reply:
i fixed this on @ayasalam0 issue it should be fixed once i merge hers